src/dune_file.ml

```diff
@@ -1014,6 +1014,23 @@ module Library = struct
             It is inherited from the virtual library."
        | _, _, _ -> ()
        end;
+       let self_build_stubs_archive =
+         let loc, self_build_stubs_archive = self_build_stubs_archive in
+         let err =
+           match c_names, cxx_names, self_build_stubs_archive with
+           | _, _, None -> None
+           | (_ :: _), _, Some _ -> Some ("c_names", loc)
+           | _, (_ :: _), Some _ -> Some ("cxx_names", loc)
```

Comment:
Why do we return the loc? It's the same in all clauses.

Reply:
Indeed, it was left-over from previous code. Fixed.